docs/reference/search/multi-search.asciidoc

```diff
@@ -86,11 +86,10 @@ the maximum number of concurrent searches the multi search api will execute.
 This default is based on the number of data nodes and the default search thread pool size.
 
 The request parameter `max_concurrent_shard_requests` can be used to control the
-maximum number of concurrent shard requests the each sub search request will execute.
+maximum number of concurrent shard requests the each sub search request will execute per node.
```

Comment:
nit: s/maximum number of concurrent shard requests the/maximum number of concurrent shard requests that/